Sources/ConsoleKit/Activity/ActivityBar.swift

```diff
@@ -35,12 +37,12 @@ extension ActivityBar {
 }
 
 /// Defines the width of all `ActivityBar`s in characters.
-private var _width: Int = 25
+private let _width: ManagedAtomic<Int> = ManagedAtomic(25)
```

Comment:
An atomic is the wrong primitive here; it doesn't yield the correct semantics. Mind you, the entire way this API works is so appallingly wrong that I'm tempted to say "let's break semver-major rules just this once to fix it" - if we take the value from the `userInfo` of the `Console` given in `outputActivityIndicator()`, we avoid this whole mss; the only caveat to doing that is it's not technically a "global" value anymore and the API contract can't be correctly maintained; we could technically avoid the semver-major break by doing this:
```swift
extension ActivityBar {
    @available(*, deprecated, message: "This value has no effect. Use `console.userInfo[\"activityBarWidth\"]` instead.")
    public static var width: Int {
        get { 25 } // deliberately hardcoded value
        nonmutating set { } // deliberately ignore new value
    }
}
```